tests/expected_data/1fas-ion.in

```diff
@@ -0,0 +1,28 @@
+read
+	mol pqr 1fas.pqr
+end
+elec
+	mg-auto
+	dime 129 97 97
+	cglen 63.1108 45.73 61.4601
+	fglen 57.124 45.73 56.153
+	cgcent mol 1
+	fgcent mol 1
+	ion charge 1 conc 0.15 radius 0.95
```

Comment:
APBS can't distinguish between ions with different radii. Can we please use 2.0 for both ion radii?

Reply:
The test file's been updated to use `2.0` for this radius